docs_user/modules/proc_rolling-back-the-control-plane-adoption.adoc

```diff
@@ -169,20 +158,12 @@ while oc get pod | grep openstack-galera-0; do
     sleep 2
 done
 
-oc delete --ignore-not-found=true --wait=false pod mariadb-copy-data
-oc delete --ignore-not-found=true --wait=false pvc mariadb-data
-oc delete --ignore-not-found=true --wait=false pod ovn-copy-data
-oc delete --ignore-not-found=true secret osp-secret
+$ oc delete --ignore-not-found=true --wait=false pod mariadb-copy-data
+$ oc delete --ignore-not-found=true --wait=false pvc mariadb-data
+$ oc delete --ignore-not-found=true --wait=false pod ovn-copy-data
+$ oc delete --ignore-not-found=true secret osp-secret
 ----
 
 [NOTE]
-Since restoring the source control plane services, their internal
-state may have changed. Before retrying the adoption procedure, it is
-important to verify that the control plane resources have
-been removed and there are no leftovers which could affect the
-following adoption procedure attempt. Notably, the previously created
-copies of the database contents must not be used in another adoption
-attempt, and new copies must be made according to the adoption
-procedure documentation.
-
-//kgilliga: How do users verify that the control plane resources have all been removed?
+After you restore the {OpenStackShort} control plane services, their internal
+state might have changed. Before you retry the adoption procedure, verify that all the control plane resources are removed and that there are no leftovers which could affect the following adoption procedure attempt. You must not use previously created copies of the database contents in another adoption attempt. You must make new copies. For more information about making new copies, see xref:migrating-databases-to-the-control-plane_storage-requirements[Migrating databases to the control plane].
```

Comment:
what is a "new copy"? Like a copy of a copy? Or a new DB instance? Maybe end user will know

Reply:
New copies of the database contents. So yes, I assume a new copy of the DB instance. @jistr Can you confirm?

Reply:
Not a copy of a copy. A copy of the original source DB taken more recently, because the content might have changed. (After we restore the original services on the source control plane, the DB there starts to get modified again. When re-trying adoption, we need to copy the latest state of the source DB contents.)